zcash_primitives/src/transaction/fees/zip317.rs

```diff
@@ -182,7 +182,7 @@ impl super::FeeRule for FeeRule {
 
         let t_out_total_size = transparent_output_sizes.into_iter().sum();
 
-        let ceildiv = |num: usize, den: usize| (num + den - 1) / den;
+        let ceildiv = |num: usize, den: usize| num.div_ceil(den);
 
         let logical_actions = max(
             ceildiv(t_in_total_size, self.p2pkh_standard_input_size),
```

Comment:
This could be just `t_in_total_size.div_ceil(self.p2pkh_standard_input_size)`, and similarly on the next line. It doesn't really matter, but a good clean-up if you have time.

Reply:
Oh, `ceildiv` only existed in the first place because it wasn't supported in the standard library at the MSRV we were on; bumping to 1.81 made this change possible and I just didn't notice.